### PR TITLE
Action copy scrubbed

### LINKED ISSUE #989: ACTION branding copy scrubbed from app
## Issue - Enhancement

Until we have another cycle with branding, we are calling the app “Parabol” and not “Parabol Action” or “Action” [PR coming soon].

**Estimated effort**: 11 points ([see CONTRIBUTING.md](https://github.com/ParabolInc/action/blob/master/CONTRIBUTING.md#points-and-sizes))

### PR DESCRIPTION
**Pull request by ackernaut**
This will fix #989

### CODE REVIEW

File: src/universal/modules/landing/containers/Landing/LandingContainer.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ export default class LandingContainer extends Component {
         APP_UPGRADE_PENDING_FALSE);
       dispatch(showInfo({
         title: 'New stuff!',
-        message: 'Action has been upgraded, log in to see what\'s new.',
+        message: 'Parabol Beta has been upgraded, log in to see what’s new.',
```
Comment (mattkrick) [inline src/universal/modules/landing/containers/Landing/LandingContainer.js]: why do we say Beta here?

File: src/universal/validation/templates.js [hunk 2]
```diff
@@ -77,20 +77,19 @@ export const orgRejectionReason = (value) => value
...
-  .min(2, 'That doesn\'t seem like much of a task')
+  .min(2, 'That doesn’t seem like much of a task')
   .max(PROJECT_MAX_CHARS, 'Try shortening down the task name');
 
 export const teamName = (value) => value
   .trim()
-  .required('"The nameless wonder" is better than nothing')
-  .min(2, 'The "A Team" had a longer name than that')
-  .max(50, 'That isn\'t very memorable. Maybe shorten it up?');
+  .required('“The nameless wonder” is better than nothing')
```
Comment (mattkrick) [inline src/universal/validation/templates.js]: the autofixed double quotes always show the ending doublequote.

File: src/server/graphql/models/Team/createFirstTeam/addSeedProjects.js [hunk 3]
```diff
@@ -15,15 +15,15 @@ const SEED_PROJECTS = [
     status: ACTIVE,
     sortOrder: 1,
     content: `
-      Try a test run of an Action meeting. Tap on ‘Meeting Lobby’ in
```
Comment (mattkrick) [inline src/server/graphql/models/Team/createFirstTeam/addSeedProjects.js]: OOoo, this is a tricky one. This isn't referring to Action, but an _Action Meeting_-- a special kind of meeting.  I kinda think we should keep this for the holacracy crowd AHA moment.

File: src/universal/validation/templates.js (hunk 2 above)
Comment (ackernaut) [inline src/universal/validation/templates.js]: what exactly do you mean? do I need to change something?

Comment (mattkrick) [inline src/universal/validation/templates.js]: Yeah, if it's important to move away from ", then we should use both “ and ”. not just ”

Comment (ackernaut) [inline src/universal/validation/templates.js]: ok, I was intentional on updating with both opening and closing, but perhaps I messed up and/or was trumped by the auto-editing (I understand what you mean now)

I’ll go back and double check, thanks for catching that!

And yes, with any copy visible to humans, my preference is to use accurate punctuation.

Comment (mattkrick) [inline src/universal/validation/templates.js]: oh nevermind, it just looks weird in the mono-spacing. the unicode value is correct. carry on!

**Comment by ackernaut** [comment]
Ok scrubbed for `“` and `”` and they seem to be in their respective places (opening/closing)…

**Comment by jordanh** [comment]
@ackernaut does all this need is a test cycle?

**Comment by ackernaut** [comment]
It’s really just copy. We cut “Action” except for “Action Meeting” the Thang™ and are using Parabol as the main branding. I also updated UI copy with “real” punctuation. We could fix anything with a patch quickly.

**Comment by ackernaut** [comment]
@jordanh once we are all clear on tests I have no further changes here…unless there are any more suggestions.

**Comment by mattkrick** [comment]
LGTM :+1: @jordanh feel free to merge when ready!

**Pull request merged**